lms/djangoapps/ora_staff_grader/serializers.py

```diff
@@ -144,14 +144,18 @@ class InitializeSerializer(serializers.Serializer):
     courseMetadata = CourseMetadataSerializer()
     oraMetadata = OpenResponseMetadataSerializer()
     submissions = serializers.DictField(child=SubmissionMetadataSerializer())
-    rubricConfig = RubricConfigSerializer()
+
+    def to_representation(self, instance):
+        """ Move rubric config inside of ORA metadata """
```

Comment:
I'm very hesitant to manually touch to_representation, and there is a way to make this serializer-y, but it's kinda ugly.

if you make 145 `oraMetadata = OpenResponseMetadataSerializer(source='*')` the it will pass "this" object to that serializer. 

Then we can just add a field to that serializer 
  `rubricConfig = RubricConfigSerializer()`
  but then would have to modify the existing field lookups to go one additional level down (`name = serializers.CharField(source='display_name')` becomes     `name = serializers.CharField(source='oraMetadata.display_name')` for example)

Reply:
I think the "right" way to do this is to transform the rubric info right from the ORA to remove having to call the rubric API (but we were leveraging the fact that the rubric API already creates the shape and (mostly) the names we wanted).

Any other way is kind of gross because this info is nested differently than we want, your method is just slightly less gross.

Reply:
We could move the rubric to the context and then have a MethodField in orametadata that just pulls that out of context?

Reply:
I don't know I think it makes most sense to have it separate and that this should be a frontend change but whatever